### ISSUE TITLE
[FEATURE]: Provide a function for setting default Canary and Production in ready to deploy when first task achieve level 2

### ISSUE THREAD

**Issue by ckw-polyu** [COMP3122-2324-Project/Simpsyber#6]
Start form level 2 task, each time the game will show Canary and Production before deploy, and the default Canary and Production is 70% and 30% which is not I want. Therefore, I need to change it every time. 
<img width="958" alt="Screenshot 2024-02-13 at 19 31 55" src="https://github.com/COMP3122-2324-Project/Simpsyber/assets/156878707/20a71673-df85-4656-adad-e4e396cbd8b4">

The game can add the function button in the READY TO DEPLOY label upper side which is similar to  Test Automation button 
<img width="185" alt="Screenshot 2024-02-13 at 20 16 30" src="https://github.com/COMP3122-2324-Project/Simpsyber/assets/156878707/11b90f26-5e6e-4794-bde5-ed43e8653b5b"> 
which look like this 
<img width="185" alt="Group 23" src="https://github.com/COMP3122-2324-Project/Simpsyber/assets/156878707/e7502408-c621-47f8-ab39-ed37d0895ad4"> 
Also, the setting will allow us to input the number instead of using button to increase and decrease.

**Comment by less2771** [comment]
Agree. I also suggest that the design could be changed to range slider, which is suitable for the function.

**Comment by IndigoNg** [comment]
I agree with (@ckw-polyu) your view, adding a settings button that allows us to enter numbers instead of using buttons to increase and decrease is indeed a good idea. In addition, I think the game system could also add a function that automatically records the last "Canary" and "Production" data. When players need to allocate the "Canary" and "Production" parameters again, the "Canary" and "Production" data will continue to display the last data. Therefore, players will not need to modify the "Canary" and "Production" data again, and this method can also enhance the player's gaming experience.

**Comment by alvinlor2001** [comment]
Agreed. I also suggest allowing players to input numbers directly from their keyboards. This will give them more options to select accurate and precise number according to their preferences.
